pkg/test/framework/components/istio/operator.go

```diff
@@ -217,90 +214,152 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
+					if err := deployControlPlane(i, cfg, cluster, remoteIopFile); err != nil {
+						return fmt.Errorf("failed deploying control plane to cluster %s: %v", cluster.Name(), err)
+					}
+					return nil
 				})
 			}
 		}
-	}
-
-	if errs := errG.Wait(); errs != nil {
-		return nil, fmt.Errorf("%d errors occurred deploying control plane clusters: %v", errs.Len(), errs.ErrorOrNil())
-	}
+		if errs := errG.Wait(); errs != nil {
+			return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
+		}
 
-	// Wait for all of the control planes to be started before deploying remote clusters
-	for _, cluster := range env.KubeClusters {
-		if env.IsControlPlaneCluster(cluster) {
-			if err := waitForControlPlane(ctx, i, cluster, cfg); err != nil {
+		if env.IsMulticluster() && !isCentralIstio(env, cfg) {
+			// For multicluster, configure direct access so each control plane can get endpoints from all
+			// API servers.
+			if err := configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
 				return nil, err
 			}
 		}
-	}
 
-	// Deploy Istio to remote clusters
-	errG = multierror.Group{}
-	for _, cluster := range env.KubeClusters {
-		if !env.IsControlPlaneCluster(cluster) {
-			cluster := cluster
-			errG.Go(func() error {
-				if err := deployControlPlane(i, cfg, cluster, remoteIopFile); err != nil {
-					return fmt.Errorf("failed deploying control plane to cluster %s: %v", cluster.Name(), err)
+		if env.IsMultinetwork() {
+			// enable cross network traffic
+			for _, cluster := range env.KubeClusters {
+				if err := createCrossNetworkGateway(ctx, cluster, cfg); err != nil {
+					return nil, err
 				}
-				return nil
-			})
+			}
 		}
-	}
-	if errs := errG.Wait(); errs != nil {
-		return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
-	}
 
-	if env.IsMulticluster() && !isCentralIstio(env, cfg) {
-		// For multicluster, configure direct access so each control plane can get endpoints from all
-		// API servers.
-		if err := configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
-			return nil, err
+		// Wait for all of the control planes to be started.
+		for _, cluster := range env.KubeClusters {
+			// TODO in centralIstiod case, webhook is only installed in control plane cluster
+			if !env.IsControlPlaneCluster(cluster) && isCentralIstio(env, cfg) {
+				continue
+			}
+			if err := waitForControlPlane(ctx, i, cluster, cfg); err != nil {
+				return nil, err
+			}
 		}
 	}
 
-	if env.IsMultinetwork() {
-		// enable cross network traffic
+	// If we are under central Istiod which uses remote KUBECONFIG, we follow below install sequences:
+	// install istiod Remote + base in control cluster first.
+	// install istiod in remote cluster
+	// get istiod loadbalancer address(discoveryAddress)
+	// create istiod service and endpoints in control cluster
+	if cfg.UsingRemoteKubeConfig {
+		i.settings.UsingRemoteKubeConfig = cfg.UsingRemoteKubeConfig
```

Comment:
It's unexpected that this would be modified here. Actually, is it redundant? https://github.com/istio/istio/pull/25784/files#diff-c7207b6e4270c44b40f0be39c12bc4eeL182

Reply:
removed